src/main/scala/com/example/Registry/ReservationSchema.scala

```diff
@@ -0,0 +1,26 @@
+package com.example.Registry
+
+import slick.jdbc.JdbcProfile
+
+final case class ReservationDB(id: Int, screeningId: Int, name: String, row: Int, seatInRow: Int)
+
+object ReservationSchema extends JdbcProfile {
+
+  import api._
+
+  class ReservationTable(tag: Tag) extends Table[ReservationDB](tag, "RESERVATIONS") {
+    def id = column[Int]("ID", O.PrimaryKey, O.AutoInc)
```

Comment:
I would see here Long rather than Int (in real production system) but fot the demo purpose it's ok